### ISSUE TITLE
Disable special keys (lightspeed.nvim compatibility)

### ISSUE THREAD

**Issue by lordvidex** [ggandor/leap.nvim#223]
I've been using lightspeed for almost a year now and I've gotten used to the key combos I use to move around. The documentation has recommended migrating to leap.nvim but every now and then, there's friction with migration due to the issue below:
### Problem
The special keys are probably nice for someone new to leap, but it's not convenient for my usecases while writing code. For example, in light speed `dt<cr>` will delete everything to the end of the current line but even after adding `flit.nvim` and setting all special keys to empty strings, I couldn't make this work as expected.
TLDR; <cr> and <space> are very valid characters for my usecase that I don't want overridden, how can i configure this?

### SideNote
My motivation for migration in the first place is the ease of configuration of leap compared to lightspeed. I have always wanted to unmap `,` because lightspeed overrides my `<localleader>` key.
